.coveragerc

```diff
@@ -0,0 +1,2 @@
+[run]
```

Comment:
can this go in the coverage section of setup.cfg?

Reply:
tried that in another repo. it has it be in `.coveragerc`

Reply:
OK, if this file is re-introduced, then probably should move the coverage stuff currently in setup.cfg (which currently includes this Cython.Coverage line)

Reply:
you're right. my bad!